contrib/python-sambacc.spec

```diff
@@ -0,0 +1,66 @@
+%global bname sambacc
+# set xversion to define the default version number
+%define xversion 0.1
+# set pversion for a customized python package version string
+%{?!pversion: %define pversion %{xversion}}
+# set rversion for a customized rpm version
+%{?!rversion: %define rversion %{xversion}}
+
+
+Name:           python-%{bname}
+Version:        %{rversion}
+Release:        1%{?dist}
+Summary:        Samba Container Configurator
+
+License:        GPLv3+
+URL:            https://github.com/samba-in-kubernetes/sambacc
+# sambacc is not released yet so we're leaving off the url for now
+# once packaged and released we can update this field
+Source:         %{bname}-%{pversion}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python3-devel
+
+%global _description %{expand:
+A library and CLI tools for managing Samba servers in containers.
+}
+
+%description %_description
+
+%package -n python3-%{bname}
```

Comment:
I am not an export to rpm spec files, but I think that if you change `Name: python3-%{bname}` you will not need this `%package -n` entry in you spec file.

Reply:
I took this from the template provided in the doc referenced in the commit msg. https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/

Reply:
Interesting. There is so much new stuff there since the last time I read those docs..